massa-bootstrap/src/client.rs

```diff
@@ -170,14 +152,12 @@ fn stream_final_state_and_consensus<D: Duplex>(
                     return Err(BootstrapError::GeneralError(String::from("Slot too old")));
                 }
                 BootstrapServerMessage::BootstrapError { error } => {
-                    return Err(std::io::Error::new(std::io::ErrorKind::InvalidData, error).into())
+                    return Err(BootstrapError::GeneralError(error))
```

Comment:
Could you please double check this change. At face value, it doesn't appear correct.

Reply:
In discord discussion, I can now see this is the case:

- it successfully gets the next message in the pipeline from the server
- the message is an error message
- it takes the form of a string
- we need to return that string as an error message

...makes sense to me to make it a ::GeneralError now.

...maybe add a comment to this matching to that effect?